src/main/kotlin/no/nav/familie/tilbake/behandling/steg/Foreslåvedtakssteg.kt

```diff
@@ -53,7 +53,7 @@ class Foreslåvedtakssteg(private val behandlingRepository: BehandlingRepository
         flyttBehandlingVidere(behandlingId)
 
         // lukker BehandleSak oppgave og oppretter GodkjenneVedtak oppgave
-        håndterOppgave(behandlingId)
+        håndterOppgave(behandlingId, true)
```

Comment:
Boolsk operator for å styre logikk i en funksjon er vel litt code smell. Her ville jeg vurdert omsrkvining med to eller flere funksjoner.